infra/ansible/playbooks/batcher.yaml

```diff
@@ -0,0 +1,143 @@
+- name: Run setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: batcher
+
+- name: Run webserver playbook
+  ansible.builtin.import_playbook: webserver.yaml
+  vars:
+    host: batcher
+
+- name: Run go playbook
+  ansible.builtin.import_playbook: go.yaml
+  vars:
+    host: batcher
+
+- name: Run rust playbook
+  ansible.builtin.import_playbook: rust.yaml
+  vars:
+    host: batcher
+
+- name: Run eigenlayer-cli playbook
+  ansible.builtin.import_playbook: eigenlayer-cli.yaml
+  vars:
+    host: batcher
+
+- hosts: batcher
+  vars:
+    service: "batcher"
```

Comment:
probably not needed for this PR, but we could look into setting this variable globally so we don't have to repeat ourselves accross the first steps (run setup playbooks)